Update language on main community page (#152)
carpentries/new-website: content/community/_index.md

The Carpentries works to help institutions and individuals spread skills for data analysis, computational thinking, and research software development through building local and global communities of practice.

> We continue to be an organization primarily supported by volunteers. Our work is made possible through the efforts of our amazing workshop hosts, organizers, community champions, member organizations, and our paid staff, the Core Team.

Reviewer: ```suggestion
We continue to be an organisation primarily supported by volunteers. Our work is made possible through the efforts of our amazing workshop hosts, organisers, community champions, member organisations, and our paid staff, the Core Team.
```

## Our Volunteer Communities

The Carpentries is a community of practice composed of regional and role-related volunteer communities. Below is a list of our role-related volunteer communities. Select a community to learn more about their role at The Carpentries.

> - [Curriculum advisors](/community/curriculum_advisors/) - guide the development of existing - lessons

Reviewer: ```suggestion
- [Curriculum advisors](/community/curriculum_advisors/) - guide the development of existing lessons
```

> - [Code of Conduct Committee](/community/coc_ctte) - keep our community friendly - and welcoming

Reviewer: ```suggestion
- [Code of Conduct Committee](/community/coc_ctte) - keep our community friendly and welcoming
```

> - [Executive Council](/about-us/governance/) - make up the governance and policy portion of The Carpentries

Reviewer: ```suggestion
- [Advisory Committee and Board of Directors](/about-us/governance/) - make up the governance and policy portion of The Carpentries
```

- [Instructors](/community/instructors) - teach and sometimes organise workshops

> - [Lesson maintainers](/community/maintainers) - support existing lessons

Reviewer: ```suggestion
- [Lesson Maintainers](/community/maintainers) - support existing lessons
```

> - [Instructor Trainers](/community/instructor-trainers) - prepare our new Instructors

Reviewer: ```suggestion
- [Instructor Trainers](/community/instructor-trainers) - train and support our new Instructors
```
